package/setup.py

```diff
@@ -364,7 +365,7 @@ def dynamic_author_list():
     "Chronological list of authors" title.
     """
     authors = []
-    with open('AUTHORS') as infile:
+    with codecs.open('AUTHORS', encoding='utf-8') as infile:
```

Comment:
This might fail in that case that we can't write to our dir. Better check now. I vote for wrapping the `dynamic_author_list` call below in a `try`/`except`.